packages/react-radio/etc/react-radio.api.md

```diff
@@ -96,6 +91,10 @@ export const useRadioGroupStyles_unstable: (state: RadioGroupState) => void;
 // @public
 export const useRadioStyles_unstable: (state: RadioState) => void;
 
+// Warnings were encountered during analysis:
+//
+// dist/packages/react-radio/src/components/Radio/Radio.types.d.ts:55:5 - (ae-forgotten-export) The symbol "RadioOnChangeData" needs to be exported by the entry point index.d.ts
```

Comment:
these comments are unfortunate. Because we use no build needed DX for v9 (that's what the local:build alias does), the output path is different thus this comment will be different on ci. to make it work you'd need to build the whole dep tree of radio via `yarn lage build --to @flunetui/react-radio`.

I did that for you so you can apply this suggestion and pipeline should pass. sorry for the inconvenience.

```suggestion
// lib/components/components/Radio/Radio.types.d.ts:55:5 - (ae-forgotten-export) The symbol "RadioOnChangeData" needs to be exported by the entry point index.d.ts
```